types/resize-observer-browser/index.d.ts

```diff
@@ -9,16 +9,33 @@ interface Window {
     ResizeObserver: typeof ResizeObserver;
 }
 
+interface ResizeObserverObserveOptions {
+    /**
+     * Sets which box model the observer will observe changes to. Possible values
+     * are `content-box` (the default), and `border-box`.
+     *
+     * @default "content-box"
+     */
+    box?: 'content-box' | 'border-box';
```

Comment:
More details of it in https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/ResizeObserver